internal/ansi/progress.go

```diff
@@ -0,0 +1,29 @@
+package ansi
```

Comment:
Any way to throw a unit test at this for coverage?

Reply:
I'll give it a go

Reply:
I can't figure out a clean way to add a unit test to this yet. It's for display purposes. I might be able to test the output, but it would be very brittle and is tied responsively to the width of the current TTY.